app/models/user.rb

```diff
@@ -228,6 +228,15 @@ def post_successful_login
     User.current = self
   end
 
+  def set_locations_and_organizations_from_usergroups(usergroups)
+    Taxonomy.enabled_taxonomies.each do |taxonomy|
+      new_taxonomies = usergroups.map{|i| i.send(taxonomy)}.flatten - send(taxonomy)
```

Comment:
Add `self` to this send or remove it from the other self below, just want to keep the method consistent.